readme.md

```diff
@@ -1,151 +1,136 @@
-- Uber
-- Buildkite
+- GitHub Cloud: Install the [Renovate Cloud-Hosted App](https://github.com/apps/renovate) on your GitHub org, then select the repos to enable
+- Bitbucket Cloud: Add the [Mend App](https://marketplace.atlassian.com/apps/1232072/mend) to your Workspace, then add the Mend Renovate user to the projects you want to enable
 
-</details>
+### Mend Renovate Community (Self-hosted)
 
-## Renovate OSS Insights
+**Supports: GitHub, GitLab, Bitbucket Data Center**
 
-Renovate is built on a big community and actively invites and supports contributions.
-Information about our contributors and community can be found on [OSS Insight](https://ossinsight.io/analyze/renovatebot/renovate).
+Install and run your own Renovate server.
+Access internal packages.
 
-## Star History
+- [Mend Renovate Community Self-Hosted](https://github.com/mend/renovate-ce-ee/tree/main/docs) (Free)
+- [Mend Renovate Enterprise](https://www.mend.io/mend-renovate/) (Paid plan)
 
-[![Star History Chart](https://api.star-history.com/svg?repos=renovatebot/renovate&type=Date)](https://star-history.com/#renovatebot/renovate&Date)
+### Other ways to run Renovate
 
-## The Renovate Approach
+If you can’t use a pre-built job scheduling system, or want to build your own, the following options are available:
 
-We believe everyone benefits from automation, whether it's a little or a lot.
-This means that Renovate:
+#### Run Renovate on your Pipeline
 
-- Adapts to your workflow
-- Allows you to configure its behavior
-- Will autodetect settings where possible
+Mend provides a _**GitHub Action**_ and _**GitLab Runner**_ to help you run Renovate as a CI pipeline job.
 
-## Using Renovate
+- GitHub Action: [renovatebot/github-action](https://github.com/renovatebot/github-action).
+- GitLab Runner: [Renovate Runner project](https://gitlab.com/renovate-bot/renovate-runner/)
+- AzureDevOps action: [Renovate Me extension](https://marketplace.visualstudio.com/items?itemName=jyc.vsts-extensions-renovate-me)<br>
+  _Note: This extension is created and maintained personally by a Renovate developer/user. Support requests for the extension will not be answered directly in the main Renovate repository._
+- Custom pipeline: You can create a custom pipeline with a **yml** definition that triggers **npx renovate**. [More details on how to configure the pipeline](https://docs.renovatebot.com/modules/platform/azure/).
 
-Get started with Renovate by checking out our [tutorial](https://github.com/renovatebot/tutorial).
+#### Run Renovate CLI
 
-### GitHub
+There are several ways to run the Renovate CLI directly.
+See docs: [Running Renovate](https://docs.renovatebot.com/getting-started/running/) for all options.
 
-We recommend that you use the Mend Renovate App.
-Install [the Mend Renovate App](https://github.com/apps/renovate) now.
-[More details on the Mend Renovate App installation](https://docs.renovatebot.com/getting-started/installing-onboarding/).
+**Targets: all platforms**
```

Comment:
```suggestion
**Supports: all platforms**
```

All other similar sentences use `Supports`. So maybe this sentence should match that pattern too?

Reply:
Thanks